src/EventHub/EventHub/help/Set-AzEventHubNamespace.md

```diff
@@ -12,20 +12,11 @@ Updates the specified Event Hubs namespace.
 
 ## SYNTAX
 
-### NamespaceParameterSet (Default)
-```
-Set-AzEventHubNamespace [-ResourceGroupName] <String> [-Name] <String> [[-Location] <String>]
- [[-SkuName] <String>] [[-SkuCapacity] <Int32>] [[-Tag] <Hashtable>] [-EnableKafka] [-DisableLocalAuth]
- [-IdentityType <String>] [-IdentityId <String[]>] [-EncryptionConfig <PSEncryptionConfigAttributes[]>]
- [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
-```
-
-### AutoInflateParameterSet
 ```
 Set-AzEventHubNamespace [-ResourceGroupName] <String> [-Name] <String> [[-Location] <String>]
  [[-SkuName] <String>] [[-SkuCapacity] <Int32>] [[-Tag] <Hashtable>] [-EnableAutoInflate]
  [-MaximumThroughputUnits <Int32>] [-EnableKafka] [-DisableLocalAuth] [-IdentityType <String>]
- [-IdentityId <String[]>] [-EncryptionConfig <PSEncryptionConfigAttributes[]>]
+ [-IdentityId <String[]>] [-EncryptionConfig <PSEncryptionConfigAttributes[]>] [-MinimumTlsVersion <String>]
```

Comment:
Please refresh this markdown, SupportsScalling is not included in current version.

Reply:
SupportsScaling is given only in cluster cmdlets and not eventhub namespace

Reply:
I will add SupportsScaling in the cluster help files. Thanks for pointing this out.